packages/pluggableWidgets/timeline-web/src/Timeline.editorPreview.tsx

```diff
@@ -0,0 +1,59 @@
+import { TimelinePreviewProps } from "../typings/TimelineProps";
+import TimelineComponent from "./components/TimelineComponent";
+import { createElement, ReactNode } from "react";
+import { ItemType } from "./Timeline";
+
+declare function require(name: string): string;
+
+export function preview(props: TimelinePreviewProps) {
+    const structuredEvents = () => {
+        const eventsMap = new Map<string | ReactNode, ItemType[]>();
+
+        Array.from({ length: 5 }).forEach(() => {
+            let constructedItem: ItemType;
+            const eventTime = props.eventTime;
+            const date = eventTime;
+            if (!date) {
+                return;
+            }
+
+            if (props.renderMode === "basic") {
+                constructedItem = {
+                    icon: props.icon,
+                    title: props.title,
+                    date: eventTime,
+                    time: eventTime,
+                    description: props.description
+                };
+            } else {
+                constructedItem = {
+                    icon: props.customIcon,
+                    dayDivider: props.customDayDivider,
+                    title: props.customTitle,
```

Comment:
This content properties needs to be handled differently for preview, we need to use `<props.cstomTitle.renderer><div /><props.cstomTitle.renderer />` -> Please follow https://github.com/mendix/widgets-resources/blob/2871d00fb5cdbbd1f28138480800433ba1426527/packages/pluggableWidgets/datagrid-web/src/Datagrid.editorPreview.tsx#L42